backend/box/box.go

```diff
@@ -682,6 +692,26 @@ func (f *Fs) List(ctx context.Context, dir string) (entries fs.DirEntries, err e
 			}
 			entries = append(entries, o)
 		}
+
+		f.itemMetaCacheMu.Lock()
+		if f.itemMetaCache != nil {
+			// If polling is enabled, cache some metadata for this Item
+			// to help us process events later on. In particular, the box
+			// event API does not provide the old path of the Item when it
+			// is renamed/deleted/moved/etc.
+			newSequenceID, err := strconv.ParseInt(info.SequenceID, 10, 64)
+			if err != nil {
+				iErr = fmt.Errorf("failed to parse `sequence_id` (%q) while listing directory: %w", info.SequenceID, err)
+				return true
```

Comment:
This exits the function with the `f.itemMetaCacheMu` locked - the suggestion above to get the JSON parser to `int` the number will fix this though.

Reply:
Done.